src/component/DisplayRecipe.js

```diff
@@ -1,17 +1,20 @@
-import { useState, useEffect } from "react";
+import { useState, useEffect, useContext } from "react";
 import RecipeDetails from "./RecipeDetails";
 import RecipeIngredients from "./RecipeIngredients";
 import RecipeDirection from "./RecipeDirection";
+import SelectedRecipeContext from "../context/SelectedRecipeContext";
 
-function DisplayRecipe({ selectedRecipe }) {
-  const [updatedServings, setUpdatedServings] = useState(
-    selectedRecipe.servings
+function DisplayRecipe() {
+  const { selectedRecipe } = useContext(SelectedRecipeContext);
+  const [selectedServings, setSelectedServings] = useState(
+    selectedRecipe?.servings || null
```

Comment:
Should `selectedServings` default value be `0` instead, since this is always a positive integer 🤔